lib/components/victory-container.js

```diff
@@ -112,7 +112,7 @@ export default class extends VictoryContainer {
       : this.panResponder.panHandlers;
     return (
       <View
-        {...handlers}
+        // {...handlers}
```

Comment:
@nicollasmatheus check out line 110. With the `disableContainerEvents` prop set, this change should not be necessary.

Reply:
p.s. to use this prop, you would do something like:
```
<VictoryChart
  containerComponent={<VictoryContainer disableContainerEvents />}
>
...
```